blazor/rating/templates.md

```diff
@@ -9,17 +9,17 @@ documentation: ug
 
 # Templates in Blazor Rating Component
 
-The Blazor Rating component supports to customize the appearance of the rating item by using Templates.
+The Blazor Rating component allows you to customize the appearance of the rating items using templates. You can use templates to specify a custom layout for the rating items, which can include any content you want. This allows you to create a more customized and interactive rating experience for the user.
 
-The Template types of Rating are as follows:
+The template types of rating are as follows:
 
-* Empty Template
-* Full Template
+* Empty template
+* Full template
 
-## EmptyTemplate
-You can access information about the un-rated item by setting [EmptyTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_EmptyTemplate) tag directive. 
+## Empty template
 
-Below example demonstrates the Empty Template of Rating.
+You can customize the appearance of the **unrated** rating items by using the [EmptyTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_EmptyTemplate) tag directive. You can use the `EmptyTemplate` tag directive to specify a custom layout for the unrated rating items, which can include any content you want. 
```

Comment:
To customize the appearance of unrated items in the Syncfusion Blazor rating component, you can use the EmptyTemplate tag directive. This directive allows you to specify a custom layout for the unrated items, which can include any content you desire.